cartography/intel/aws/ec2/_init_.py

```diff
@@ -0,0 +1,26 @@
+import logging
+
+from intel.aws.ec2 import auto_scaling_groups
+from intel.aws.ec2 import instances
+from intel.aws.ec2 import key_pairs
+from intel.aws.ec2 import load_balancers
+from intel.aws.ec2 import security_groups
+from intel.aws.ec2 import vpc
+from intel.aws.ec2 import vpc_peering
```

Comment:
I'd love to make these `from . import …` but the mypy pre-commit check fails with `error: No parent module -- cannot perform relative import`.